app/src/main/java/com/infomaniak/swisstransfer/ui/screen/newtransfer/importfiles/components/PasswordOptionAlertDialog.kt

```diff
@@ -102,13 +100,17 @@ private fun ColumnScope.AnimatedPasswordInput(
     isPasswordValid: () -> Boolean
 ) {
     AnimatedVisibility(isChecked) {
-        Spacer(Modifier.height(Margin.Mini))
         SwissTransferTextField(
+            modifier = Modifier.fillMaxWidth(),
             label = stringResource(R.string.settingsOptionPassword),
             isPassword = true,
             initialValue = password.get(),
             imeAction = ImeAction.Done,
-            errorMessage = { if (isPasswordValid()) null else stringResource(R.string.errorTransferPasswordLength) },
+            isError = !isPasswordValid(),
+            supportingText = { isError ->
+                val errorText = if (isError) stringResource(R.string.errorTransferPasswordLength) else ""
+                Text(text = errorText, minLines = 2)
```

Comment:
When going the "alpha route" minLines won't be needed anymore, I just add this here but you might have already modified it